litellm/main.py

```diff
@@ -98,17 +98,23 @@ class Chat():
   def __init__(self, params):
     self.params = params
     self.completions = Completions(self.params)
-
+      
 class Completions():
   
-  def __init__(self, params):
+  def __init__(self, model, params):
```

Comment:
Hey @estill01 the pr would break the LiteLLM + Router classes which use Completions to provide similar functionality to OpenAI().completion.create

Reply:
I believe the issue is fixed; you can do e.g. `LiteLLM(model='gpt-4')` and it will persist it; gets picked up in `self.params`

Reply:
but what if i don't set model in the class? i don't think this is a pattern that makes sense within litellm where people specify model on completion call

Reply:
@estill01 could we make this an optional param? This way the existing clients aren't impacted, and your use-case would still be supported